app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class UserMailer < ApplicationMailer
+  def activation_needed_email(user)
```

Comment:
It is the default sorcery name for this email, but i see i can change it in the configs will do :+1:!